scalar/timestamp.go

```diff
@@ -18,6 +18,7 @@ const defaultStringFormat = "2006-01-02 15:04:05.999999999 -0700 MST"
 
 // this is used by arrow string format (time is in UTC)
 const arrowStringFormat = "2006-01-02 15:04:05.999999999"
+const arrowStringFormatNew = "2006-01-02 15:04:05.999999999Z"
```

Comment:
Not adding a new line and using the one above seems to break some other things (at least in the test src plugin)